test_communication/test/message_fixtures.py

```diff
@@ -98,17 +98,18 @@ def get_msg_primitives():
     msg.bool_value = True
     msg.byte_value = bytes([1])
     msg.char_value = '\x01'
-    msg.float32_value = float(1.125)
-    msg.float64_value = float(1.125)
-    msg.int8_value = int(1)
-    msg.uint8_value = int(1)
-    msg.int16_value = int(1)
-    msg.uint16_value = int(1)
-    msg.int32_value = int(1)
-    msg.uint32_value = int(1)
-    msg.int64_value = int(1)
-    msg.uint64_value = int(1)
+    msg.float32_value = float(1.0)
+    msg.float64_value = float(1.0)
```

Comment:
I would rather check for an actual float value.

Isn't C++ using 1.125? https://github.com/ros2/system_tests/pull/161/files#diff-d88127361752475ebfb4ee2872e5bc4bR70